src/tests/JIT/Stress/ABI/pinvokes_d.csproj

```diff
@@ -1,14 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
+    <!-- Needed for GCStressIncompatible, JitOptimizationSensitive -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+
+    <!-- Main supports command-line arguments -->
+    <ReferenceXUnitWrapperGenerator>false</ReferenceXUnitWrapperGenerator>
+    <CLRTestExecutionArguments>--pinvokes --num-calls 1000 --no-ctrlc-summary</CLRTestExecutionArguments>
```

Comment:
I found this kind of test frustrating to convert.  I'd like to have something like

```
[Theory]
[InlineData("--pinvokes ...")]
public static test(string[] args)

// somehow not interfering with ReqProcIso/BuildAsStandalone
public static void Main(string[] args)
```

OR have the test have symbolic inputs (`bool pinvokes`) and only use the parser for Main.  But that's a lot of work so probably too much (not just for this PR but in general).

Reply:
I completely agree there are many tests that can be improved in a similar vein. As I mentioned before, I still believe that the most immediate value can be had by just finishing the test merging process and disposing of the legacy infrastructure but I'm definitely looking forward to further cleanups in our vast test portfolio.